### PR TITLE
Remove lumberjack

### PR DESCRIPTION
**Pull request by LGriffioen**

### CODE REVIEW

**Comment by roballie** [comment]
Did something go wrong in the rebase?  Why is there a project file, info plist and blank umbrella header from the framework in the first commit?

**Comment by LGriffioen** [comment]
hm, seems like maybe Dave had the same idea as you on the project file, but put it in another directory?

**Comment by roballie** [comment]
ohhh... that makes sense.  I was so confused.  Didn't see that Dave made those files.

**Comment by roballie** [comment]
My build fails.  Reference to missing file "HTTPLogging.h" in "MultipartMessageHeader.m"

**Comment by LGriffioen** [comment]
yeah, seems like this should have been merged ages ago. i also noticed there's 2 other snagit fix warnings branches, but i'm probably just going to delete those since there's crazy conflicts

**Comment by LGriffioen** [comment]
right, the idea here was to get rid of cocoa lumberjack

**Comment by roballie** [comment]
But building CocoaHTTPServer alone fails.  not my parent app build.

**Comment by LGriffioen** [comment]
oh i see, looks like there's new classes since this was done that have the logging junk

**Comment by roballie** [comment]
I can help tackle this if you want.  Probably easier to just delete the project that Dave made.

**Comment by LGriffioen** [comment]
i think i figured it out

File: CocoaHTTPServer/CocoaHTTPServer.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -485,7 +485,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.15;
+				MACOSX_DEPLOYMENT_TARGET = 10.13;
```
Comment (roballie) [inline CocoaHTTPServer/CocoaHTTPServer.xcodeproj/project.pbxproj:399]: oops, I thought I had changed this.  Must have lost the change at some point.

Comment (roballie) [reply CocoaHTTPServer/CocoaHTTPServer.xcodeproj/project.pbxproj:399]: oh, I see.  I had it in the framework itself set at 10.9 to silence the deprecations

**Comment by LGriffioen** [comment]
should i take the time now to upgrade all of the commented out log statements to `os_log`?

**Comment by roballie** [comment]
Up to you.  Probably a separate PR.

**Comment by LGriffioen** [comment]
i guess i was wondering if mobile folks rely of those logs at all. i'm not sure we've ever looked at them on snagit

**Comment by LGriffioen** [comment]
nevermind, this is a macOS only framework. derp

**Review by roballie — APPROVED** [review]
Looks 👍 to me... but what if I liked cocoaLumberjack :trollface:

**Comment by LGriffioen** [comment]
it wasn't bad before os_log

**Comment by LGriffioen** [comment]
@hoolrory I assume Camtasia uses this. any thoughts?

**Comment by hoolrory** [comment]
Hmmm we removed CocoaLumberjack from Camtasia last year
It looks like we're only using this submodule via TSCConnectServer which pulls in some of the source code to TSCConnectServer.framework

**Comment by LGriffioen** [comment]
you might want to grab this branch then, Rob did some work on TSCConnectServer so it no longer includes this library, you have to add it to your parent app

**Comment by LGriffioen** [comment]
@hoolrory did you get a chance to try this out? i want to make sure it doesn't make life harder for you before merging it

**Comment by hoolrory** [comment]
@LGriffioen I updated to latest TSCConnectServer, then pointed to this branch. Then I added the CocoaAsyncSocket and CocoaHTTPServer projects to the Xcode workspace, but I'm still having issues:
image

@roballie any ideas, is there a different TSCConnectServer branch I should be using?

**Comment by hoolrory** [comment]
@LGriffioen tried a build on build server to see if a clean build would work, but it had same errors

**Comment by roballie** [comment]
Did you use the TSCConnectServer framework?

**Comment by roballie** [comment]
You can look at the workspace in the test harness repo to see how it all links together.  https://github.com/TechSmith/TSCConnect-TestHarness

**Review by hoolrory — APPROVED** [review]
Works for Camtasia now, had to remove one unused import to fix the build issues, no idea what was going on there 🤷 
https://github.com/TechSmith/Camtasia-Mac/pull/6133

**Pull request merged**